api/src/main/resources/liquibase.xml

```diff
@@ -217,4 +217,20 @@
 			UPDATE global_property SET property_value = 'false' WHERE property IN ('fhir.started', 'dataentrystatistics.started');
 		</sql>
 	</changeSet>
+
+	<changeSet id="ugandaemr-030520191333" author="slubwama">
```

Comment:
@slubwama Just add this global property to the config.xml

Reply:
@ssmusoke adding it to config causes an overwrite every time the module starts thats why Its in liquibase.  For variable values Its not good to keep them in config file. I have tried that already.

Reply:
@slubwama Nope its the recommended place to define global properties in OpenMRS, its run only once